shtdlib.sh

```diff
@@ -2668,6 +2673,7 @@ function test_shtdlib {
         debug ${i} "Debug Level ${i}"
     done
     verbosity="${orig_verbosity}"
+    echo ${verbosity}
```

Comment:
Is this an old debug message, I don't think we need to echo this no?

Reply:
My fault I forgot to push a squashed commit